11ty/definitions/fr_FR/irreel.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Irréel
+slug: irreel
+defined: true
+excerpt: très impressionnant ; étonnant ; plus imposant ou plus fantastique que ce qui est typique de la vie réelle.
+speech: adj
+---
```

Comment:
Add definition reference cited by @deammer

Reply:
Reference added and the title is also in lowercase now.